test/Graphics/Shader.test.cpp

```diff
@@ -163,17 +163,51 @@ TEST_CASE("[Graphics] sf::Shader", skipShaderFullTests())
         STATIC_CHECK(std::is_nothrow_move_assignable_v<sf::Shader>);
     }
 
-    if (!sf::Shader::isAvailable())
-        return;
+    SECTION("Construction")
+    {
+        const sf::Shader shader;
+        CHECK(shader.getNativeHandle() == 0);
+    }
+
+    SECTION("loadFromFile()")
+    {
+        SECTION("One shader")
+        {
+            sf::Shader shader;
+            CHECK(!shader.loadFromFile("does-not-exist.vert", sf::Shader::Type::Vertex));
+            CHECK(shader.loadFromFile("Graphics/shader.vert", sf::Shader::Type::Vertex) == sf::Shader::isAvailable());
```

Comment:
This is testing only `Type::Vertex`. What about `Type::Fragment` and `Type::Geometry`?
If `Type::Geometry` crashes on the debian worker, maybe there is a bug to fix?

Reply:
I added a test for loading a fragment shader but couldn't get a geometry shader to load. The Linux and Windows jobs are showing that loading the geometry shader is failing in spite of geometry shaders being available.

Reply:
I think this is expected. The preexisting test about loadFromMemory expects loading a geometry shader alone fails.

Reply:
I see what you mean. It's weird that geometry shaders always fail to load though, no? Why have this API if it always fails?